src/main/java/com/spotinst/sdkjava/model/bl/ocean/aks/ImportClusterAutomatic.java

```diff
@@ -0,0 +1,77 @@
+package com.spotinst.sdkjava.model.bl.ocean.aks;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class ImportClusterAutomatic {
```

Comment:
can we rename it to ImportClusterHeadroomAutomatic. Same for API Class.

Reply:
updated for both